source/redirects.txt

```diff
@@ -111,23 +111,6 @@
 "docs/software/basic-programming/robot-code-ci.rst" "docs/software/advanced-gradlerio/robot-code-ci.rst"
 "docs/software/advanced-controls/introduction/controls-glossary.rst" "docs/software/advanced-controls/controls-glossary.rst"
 "docs/software/advanced-controls/state-space/state-space-glossary.rst" "docs/software/advanced-controls/controls-glossary.rst"
-"docs/software/old-commandbased/index.rst" "docs/software/commandbased/index.rst"
-"docs/software/old-commandbased/basics/index.rst" "docs/software/commandbased/index.rst"
-"docs/software/old-commandbased/basics/adding-commands-subsystems.rst" "docs/software/commandbased/index.rst"
-"docs/software/old-commandbased/basics/convert-simple-auto-command-auto.rst" "docs/software/commandbased/index.rst"
-"docs/software/old-commandbased/basics/creating-the-project.rst" "docs/software/commandbased/index.rst"
-"docs/software/old-commandbased/basics/what-is-command-based.rst" "docs/software/commandbased/index.rst"
-"docs/software/old-commandbased/commands/index.rst" "docs/software/commandbased/index.rst"
-"docs/software/old-commandbased/commands/creating-simple-commands.rst" "docs/software/commandbased/index.rst"
-"docs/software/old-commandbased/commands/default-commands.rst" "docs/software/commandbased/index.rst"
-"docs/software/old-commandbased/commands/running-commands-autonomous.rst" "docs/software/commandbased/index.rst"
-"docs/software/old-commandbased/commands/running-commands-joystick-input.rst" "docs/software/commandbased/index.rst"
-"docs/software/old-commandbased/commands/scheduling-commands.rst" "docs/software/commandbased/index.rst"
-"docs/software/old-commandbased/commands/creating-groups-commands.rst" "docs/software/commandbased/index.rst"
-"docs/software/old-commandbased/commands/synchronizing-two-commands.rst" "docs/software/commandbased/index.rst"
-"docs/software/old-commandbased/subsystems/index.rst" "docs/software/commandbased/index.rst"
-"docs/software/old-commandbased/subsystems/pidsubsystems-pid-control.rst" "docs/software/commandbased/index.rst"
-"docs/software/old-commandbased/subsystems/simple-subsystems.rst" "docs/software/commandbased/index.rst"
```

Comment:
redirects.txt shouldn't have lines deleted from it.

I'll PR a CI check for this later.

Reply:
I think the rediraffewritediff keeps deleting stuff?